### PR TITLE
Add cockroach-sql formula

### PR DESCRIPTION
**Pull request by rail**
Jira: RE-331

### CODE REVIEW

**Comment by rail** [comment]
The first commit is from https://github.com/cockroachdb/homebrew-tap/pull/117

File: Formula/cockroach-sql.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Auto-generated file, DO NOT EDIT
+# Source: release/cockroach-sql-tmpl.rb
+
+class CockroachSql < Formula
+  desc "Distributed SQL database shell"
+  homepage "https://www.cockroachlabs.com"
+  version "22.2.0-beta.5"
```
Comment (celiala) [inline Formula/cockroach-sql.rb:7]: Do we want pre-releases? (For other brew formulas, we currently only do this for latest stable releases) 

If so, maybe this initial formula is for v22.1.{latest-patch}.

**Review by celiala — APPROVED** [review]
Code itself LGTM, but I had a question regarding initial value for version (maybe initial version should be "22.1.{something}" instead of "22.2.0-beta.5"?)

File: Formula/cockroach-sql.rb (hunk 1 above)
Comment (rail) [reply Formula/cockroach-sql.rb:7]: Sorry, I wasn't clear regarding this. The idea is to merge this after #117, which implies 22.2.0 released. At that point I would need to update the PR and regenerate this file with 22.2.0.

Comment (celiala) [reply Formula/cockroach-sql.rb:7]: Oh, maybe [comment](https://github.com/cockroachdb/homebrew-tap/pull/117#issuecomment-1300729435) from applies here as well?

> Do not merge until 22.2.0 is stable

**Pull request merged**